py-polars/polars/datatypes/classes.py

```diff
@@ -484,6 +481,25 @@ def __repr__(self) -> str:
             f"{class_name}(time_unit={self.time_unit!r}, time_zone={self.time_zone!r})"
         )
 
+    def _validate_timezone(self, time_zone: str | timezone | None) -> str | None:
+        if time_zone is None or time_zone == "*":
+            return time_zone
+        elif isinstance(time_zone, timezone):
+            time_zone = time_zone
+        else:
+            from polars.dependencies import _ZONEINFO_AVAILABLE, zoneinfo
+
+            if _ZONEINFO_AVAILABLE:
+                if time_zone in zoneinfo.available_timezones():
```

Comment:
Does this match with the list of timezones we use on the Rust side? What are the differences if there are any?

Reply:
This is the part that concerns me a little, I'm pretty sure I encountered some differences